shared/README.md

```diff
@@ -42,9 +74,9 @@ After saving the file set environment using:
 . .env
 ```
 
-Alternatively, use the cluster.tfvars file to set parameter values as shown below:
+Alternatively, use the shared.tfvars file to set parameter values as shown below:
 
 ``` 
-location="esatus"
+location="eastus"
 company="myCompany"
 ```
```

Comment:
Missing the following variable values:
- storage_account_name
- container_name
- access_key

Reply:
Its defined in the backend tfvars info. Those two are separate. Cannot be merged.